client/src/globalState.tsx

```diff
@@ -460,8 +565,9 @@ export const GlobalStateProvider = ({
       );
     }
     setSelectedPinboardId(null);
-    setUnreadFlag(pinboardIdToClose)(undefined);
     setError(pinboardIdToClose, undefined);
+    //TODO probably need to check if pinboard is also in group pinboards, so we don't lose an unread on there
```

Comment:
is this a todo that should be addressed in this PR, or is it one for an optional followup?

Reply:
![pinboard_group_unread_after_close](https://github.com/user-attachments/assets/9d5119b4-c2d1-4227-a450-a855094b7d80)